lib/cartopy/crs.py

```diff
@@ -1815,6 +1815,59 @@ def y_limits(self):
         return self._y_limits
 
 
+class LambertZoneII(Projection):
+    """
+    Lambert zone II (extended) projection (https://epsg.io/27572).
+
+    """
+    def __init__(self):
+        proj4_params = [('proj', 'lcc'),
+                        ('lat_1', 46.8),
+                        ('lat_0', 46.8),
+                        ('lon_0', 0.0),
+                        ('k_0', 0.99987742),
+                        ('x_0', 600000.0),
+                        ('y_0', 2200000.0),
+                        ('pm', 'paris'),
+                        ('units', 'm'),
+                        ('no_defs', None)]
+
+        globe = Globe(semimajor_axis='6378249.2',
+                      semiminor_axis='6356515.0',
+                      towgs84='-168,-60,320,0,0,0,0')
```

Comment:
Where do these values come from? (I didn't see anything about this in the EPSG website) Is this projection only valid with this `Globe` and do we need to allow users to override this default value?

Reply:
This ellipsoid corresponds to [EPSG:7011](https://epsg.io/7011-ellipsoid.json). These values (and the conversion to WGS84) can be found in a [report [in French]](https://geodesie.ign.fr/contenu/fichiers/documentation/pedagogiques/TransformationsCoordonneesGeodesiques.pdf) produced by the [French National Institute of Geographic and Forest Information](https://www.ign.fr/institut/identity-card) in sections 3.1.1 and 4.3.1, respectively. As EPSG:27572 clearly defines the ellipsoid to use, I don't think we need to allow users to override them.